parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -226,7 +226,11 @@ if [[ -n ${MASTER_NODE} ]]; then
 {{if IsAzurePolicyAddonEnabled}}
   time_metric "EnsureLabelExclusionForAzurePolicyAddon" ensureLabelExclusionForAzurePolicyAddon
 {{end}}
-  time_metric "EnsureAddons" ensureAddons
+  if [ -f /var/run/reboot-required ] || [ "$NO_OUTBOUND" = "true" ]; then
```

Comment:
looking at https://github.com/Azure/aks-engine/pull/3410/files, we might actually not need this for NO_OUTBOUND @jackfrancis

Reply:
removed the no_outbound condition